### PR TITLE
modal component added

### PR DESCRIPTION
**Pull request by akash-1318**

### CODE REVIEW

File: components/css/componentStyle/Modal.css [hunk 1]
```diff
@@ -0,0 +1,182 @@
+.modal-component{
```
Comment (tushargoyal812) [inline components/css/componentStyle/Modal.css:1]: Suggestion: add space before { everywhere it is good for indentation.

**Review by tushargoyal812 — COMMENTED** [review]
everything looks good just take care of indentation i suggested.

**Comment by yogesh-** [comment]
While doing a PR put a proper comment about what have you made + any video or image to support the comments about the screen. Best practices. You can also put preview link.

File: components/css/componentStyle/Modal.css [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    background-color: #7c7c7c;
+}
+
+.dialog-box-btn{
+    padding: 1.5rem 5rem;
+    background-color: #ffffff;
+    border: 1px solid rgb(199, 199, 199);
+    border-radius: 5px;
+    cursor: pointer;
+}
```
Comment (yogesh-) [inline components/css/componentStyle/Modal.css]: either use px or rem together.
Stick to one.
Best practice is to use rem.

File: components/css/componentStyle/Modal.css [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 80%;
+    width: 30%;
+    margin: auto;
+    border: 1px solid rgb(199, 199, 199);
+    background-color: #ffffff;
```
Comment (yogesh-) [inline components/css/componentStyle/Modal.css]: use color variables

**Review by yogesh- — APPROVED** [review]
Minor changes suggested for best practices. rest all looks good

**Pull request merged**